### ISSUE TITLE
kotlin/dokka#158: New lines in Java source Javadoc code blocks are not preserved in dokka HTML output

### ISSUE THREAD

**Issue by eskatos** [gradle/kotlin-dsl#363]
Given a Java type with some Javadoc, e.g.:

```java
/**
 * Something that matters.
 *
 * <pre>
 * with( some ) {
 *     multi = lines
 *     sample()
 * }
 * </pre>
 */
public interface Something {}
```

All lines in the `<pre/>` element are rendered as a single line in dokka HTML output.

As an example:
- [Java source for Copy](https://github.com/gradle/gradle/blob/d60b527ff902fd0710f64972ad25f7ed02c289bc/subprojects/core/src/main/java/org/gradle/api/tasks/Copy.java#L29-L69)
- [Javadoc output for Copy](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Copy.html)
- [dokka output for Copy](https://gradle.github.io/gradle-script-kotlin-docs/api/org.gradle.api.tasks/-copy/index.html)

See kotlin/dokka#158

**Comment by eskatos** [comment]
Upstream issue is fixed but `dokka` still choke on our usage of html attributes on `<pre>` tags in javadoc, see e.g. https://github.com/gradle/gradle/blob/d60b527ff902fd0710f64972ad25f7ed02c289bc/subprojects/core/src/main/java/org/gradle/api/tasks/Copy.java#L34

**Comment by eskatos** [comment]
Upstream issue acknowledged and reopened.

**Comment by eskatos** [comment]
Fixed in 0.9.18, closing
